### PR TITLE
vimPlugins followup

### PR DESCRIPTION
**Pull request by teto**
###### Motivation for this change

trying to address some vim issues possibly triggered by https://github.com/NixOS/nixpkgs/pull/136429

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 137885` at b95ce9ce run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>8 packages built successfully:</summary>
  <ul>
    <li>spacevim</li>
    <li>tests.vim.test_vim_with_custom_plugin</li>
    <li>tests.vim.test_vim_with_vim_nix</li>
    <li>tests.vim.test_vim_with_vim_nix_using_pathogen</li>
    <li>tests.vim.test_vim_with_vim_nix_using_plug</li>
    <li>tests.vim.test_vim_with_vim_nix_using_vam</li>
    <li>tests.vim.vim_with_vim2nix</li>
    <li>vimacs</li>
  </ul>
</details>

**Comment by figsoda** [comment]
@ofborg eval neovim.tests

**Review by rvolosatovs — APPROVED** [review]

File: pkgs/misc/vim-plugins/vim-utils.nix [hunk 1]
```diff
@@ -187,8 +187,10 @@ let
 
   rtpPath = ".";
 
-  nativeImpl = packages:
-  (let
+  /* Generates a packpath folder as expected by vim
+  */
```
Comment (SuperSandro2000) [inline pkgs/misc/vim-plugins/vim-utils.nix:191]:
```suggestion
  # Generates a packpath folder as expected by vim
```

Comment (teto) [reply pkgs/misc/vim-plugins/vim-utils.nix]: done

**Comment by teto** [comment]
@minijackson could you check it fixes the tag generation for you ?

**Comment by teto** [comment]
this does not fix the vim-plug issue (will try again once I can reproduce the issue), but it adds some more testing, especially for tag generation.

**Pull request merged**